Slash/DB/MySQL/MySQL.pm

```diff
@@ -6295,7 +6297,7 @@ sub getCommentTextCached {
 	$opt ||= {};
 
 	my $possible_chop  = !$opt->{full} && !($opt->{mode} && $opt->{mode} eq 'archive');
-	my $abbreviate_ok  = $opt->{discussion2} && $possible_chop;
+	my $abbreviate_ok  = 0;
```

Comment:
that code makes the comment do a partial collapse iirc.  We could leave it in and do a css uncollapse on it.